src/App.js

```diff
@@ -0,0 +1,23 @@
+import "./App.css";
+import React from "react";
+import Header from "./components/Header/Header";
+import InfoSection from "./components/InfoSection/InfoSection";
+import ProjectsSection from "./components/Projects/ProjectsSection";
+
+function App() {
+  return (
+    <div className="wrapper">
+      <Header></Header>
```

Comment:
It is better to use self-closing tags, e.g. 
```suggestion
      <Header />
```